models/Adult.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
```

Comment:
🚫 **[eslint]** <[import/newline-after-import](https://github.com/import-js/eslint-plugin-import/blob/v2.25.3/docs/rules/newline-after-import.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected 1 empty line after require statement not followed by another require.
```suggestion
const mongoose = require('mongoose');

```